src/controller/gap-controller.ts

```diff
@@ -145,6 +145,11 @@ export default class GapController {
       this._reportStall(bufferInfo.len);
     }
 
+    // Handle case where destroy() has been called & it set media to null
+    if (!media) {
```

Comment:
```suggestion
    if (!this.media) {
```

we need to check `this.media` because `media` won't be effected by `this.media` being nulled

Reply:
Ah, good catch! I should've caught that. Will fix & push.